app/Models/BuildPropagationHistory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class BuildPropagationHistory extends Model
+{
+    protected $table = 'osu_build_propagation_history';
+    const UPDATED_AT = null;
```

Comment:
not sure if it's a supported construct. Perhaps better just set `$timestamps` to `false`.